uc-catalog-cloning/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (2023) Databricks, Inc.
```

Comment:
you don't need a license in this folder, it's there in the root - https://github.com/databrickslabs/sandbox/blob/main/LICENSE